draft-ietf-taps-transport-security.md

```diff
@@ -413,57 +467,76 @@ IKEv2 is a control protocol that runs on UDP port 500. Its primary goal is to ge
 IKEv2 first uses a Diffie-Hellman key exchange to generate keys for the "IKE SA", which is a set of keys used to encrypt further IKEv2 messages. It then goes through a phase of authentication in which both peers present blobs signed by a shared secret or private key, after which another set of keys is derived, referred to as the "Child SA". These Child SA keys are used by ESP.
 
-IKEv2 negotiates which protocols are acceptable to each peer for both the IKE and Child SAs using "Proposals". Each proposal may contain an encryption algorithm, an authentication algorithm, a Diffie-Hellman group, and (for IKE SAs only) a pseudorandom function algorithm. Each peer may support multiple proposals, and the most preferred mutually supported proposal is chosen during the handshake.
+IKEv2 negotiates which protocols are acceptable to each peer for both the IKE and Child SAs using 
+"Proposals". Each proposal may contain an encryption algorithm, an authentication algorithm, a 
+Diffie-Hellman group, and (for IKE SAs only) a pseudorandom function algorithm. Each peer may 
+support multiple proposals, and the most preferred mutually supported proposal is chosen during 
+the handshake.
 
-The authentication phase of IKEv2 may use Shared Secrets, Certificates, Digital Signatures, or an EAP (Extensible Authentication Protocol) method. At a minimum, IKEv2 takes two round trips to set up both an IKE SA and a Child SA. If EAP is used, this exchange may be expanded.
+The authentication phase of IKEv2 may use Shared Secrets, Certificates, Digital Signatures, or an 
+EAP (Extensible Authentication Protocol) method. At a minimum, IKEv2 takes two round trips to set 
+up both an IKE SA and a Child SA. If EAP is used, this exchange may be expanded.
 
-Any SA used by IKEv2 can be rekeyed upon expiration, which is usually based either on time or number of bytes encrypted.
+Any SA used by IKEv2 can be rekeyed upon expiration, which is usually based either on time or 
+number of bytes encrypted.
 
 There is an extension to IKEv2 that allows session resumption {{RFC5723}}.
 
-MOBIKE is a Mobility and Multihoming extension to IKEv2 that allows a set of Security Associations to migrate over different addresses and interfaces {{RFC4555}}.
+MOBIKE is a Mobility and Multihoming extension to IKEv2 that allows a set of Security Associations 
+to migrate over different addresses and interfaces {{RFC4555}}.
 
 When UDP is not available or well-supported on a network, IKEv2 may be encapsulated in TCP {{RFC8229}}.
 
 #### ESP {#ESP}
  
-ESP is a protocol that encrypts and authenticates IPv4 and IPv6 packets. The keys used for both encryption and authentication can be derived from an IKEv2 exchange. ESP Security Associations come as pairs, one for each direction between two peers. Each SA is identified by a Security Parameter Index (SPI), which is marked on each encrypted ESP packet.
```

Comment:
Is there a reason to word-wrap these lines? It potentially complicates diffs, making it more difficult to see what actually changed. I like the approach of one sentence per line: markdown will reflow them into a single paragraph, and it makes the diff meaningful in the sense that only sentences that change appear changed in the diff.

To be clear, I don't think we should go back and change it all now, just choose an approach and stick with it for future changes.

Reply:
One sentence per line is fine, and would make lines much shorter. Paragraphs on a single line make diffs a nightmare.

Reply:
The diffs in github wrap long lines, at least, but I don't have a strong opinion about this either way. I think my minimalist objection is: I would simply avoid moving around whitespace (e.g., changing the position of newlines) when editing text that will render the same way with the whitespace unmoved.